src/granite_orm/fields.cr

```diff
@@ -140,6 +140,8 @@ module Granite::ORM::Fields
           {% end %}
         {% end %}
       end
+    rescue ex
+      errors << Granite::ORM::Error.new(name, ex.message)
```

Comment:
Nice! 👍

Reply:
This code is pretty well constrained to the type casting, but if it is possible to be more specific about what is being rescued, that would probably be safer.

Reply:
@robacarp Yeah, I was hoping too as well.  I looked to see if I could catch a specific Exception but there is none provided by crystal String.  I don't want to try and determine this by the error message itself since there are many possibilities.